pkg/util/util.go

```diff
@@ -72,8 +72,14 @@ func SetupFlyteDir() error {
 	return nil
 }
 
-// PrintDemoMessage will print sandbox success message
+// PrintDemoMessage will print demo start success message
+// Deprecated: use PrintDemoStartMessage
```

Comment:
If we are confident this is not used by any potential users, it is fine to delete.

Reply:
I doubt that there's someone out there importing this. It's fine to delete.